### PR TITLE
Update Mixer FAQ answer.

### PR DESCRIPTION
**Pull request by geeknoid**

### CODE REVIEW

File: _faq/why-mixer.md [hunk 1]
```diff
@@ -10,10 +10,28 @@ and the infrastructure backends used to perform access control checks and teleme
...
-to manage scaling and reliability concerns independently . The proxy is focused on
-high-performance routing, while Mixer takes responsibility for policy management and telemetry collection.
-Mixer is invoked by Envoy, and in the future will also be invoked directly by Istio-based services.
+Mixer is designed as a stand-alone component, distinct from Envoy. This has numerous benefits:
+
+- *Scalability*.
+The work that Mixer and Envoy do is very different in nature, leading to different scalability
+requirements. Keeping the components separate enables independent component-appropriate scaling.
+
+- *Resurce Usage*.
```
Comment (mandarjog) [inline _faq/why-mixer.md]: Resource

File: _faq/why-mixer.md [hunk 2]
```diff
@@ -10,10 +10,28 @@ and the infrastructure backends used to perform access control checks and teleme
...
+The work that Mixer and Envoy do is very different in nature, leading to different scalability
+requirements. Keeping the components separate enables independent component-appropriate scaling.
+
+- *Resource Usage*.
+Istio depends on being able to deploy many instances of its proxy, making it important to minimize the
+cost of each individual instance. Moving Mixer's complex logic into a distinct component makes it 
+possible for Envoy to remain svelte and agile.
+
+- *Reliability*.
+Mixer's operation is fairly complex, making it more susceptible to failures. Separating
```
Comment (douglas-reid) [inline _faq/why-mixer.md]: i might comment on the fact that Mixer hosts 3rd-party logic and adapters that can be a source of issues.  As you note, having that integration point distinct from the proxy provides some isolation.

File: _faq/why-mixer.md [hunk 3]
```diff
@@ -10,10 +10,28 @@ and the infrastructure backends used to perform access control checks and teleme
...
+the components introduces distinct failure domains, enabling Envoy to continue
+operating even if Mixer fails.
+
+- *Isolation*. Mixer provides a level of insulation between Istio and the infrastructure backends. Each Envoy instance can be configured to have a 
+very narrow scope of interaction, limiting the impact of potential attacks.
+
+- *Extensibility*. It was imperative to design a simple extensibility model to allow Istio to interoperate
```
Comment (douglas-reid) [inline _faq/why-mixer.md]: Even more, this allows use of custom/alternate proxies without having to port policy enforcement and telemetry across systems.

File: _faq/why-mixer.md [hunk 4]
```diff
@@ -10,10 +10,28 @@ and the infrastructure backends used to perform access control checks and teleme
...
+- *Scalability*.
+The work that Mixer and Envoy do is very different in nature, leading to different scalability
+requirements. Keeping the components separate enables independent component-appropriate scaling.
+
+- *Resource Usage*.
+Istio depends on being able to deploy many instances of its proxy, making it important to minimize the
+cost of each individual instance. Moving Mixer's complex logic into a distinct component makes it 
+possible for Envoy to remain svelte and agile.
+
+- *Reliability*.
```
Comment (mandarjog) [inline _faq/why-mixer.md]: I would like to explain this in positive terms.

Mixer isolates the most complex parts of the data path in a distinct failure domain that is decoupled from Envoy, enabling Envoy to continue operating even if Mixer fails.

File: _faq/why-mixer.md (hunk 2 above)
Comment (geeknoid) [inline _faq/why-mixer.md]: Done

File: _faq/why-mixer.md (hunk 4 above)
Comment (geeknoid) [inline _faq/why-mixer.md]: Done

File: _faq/why-mixer.md (hunk 3 above)
Comment (geeknoid) [inline _faq/why-mixer.md]: Done.

**Comment by ldemailly** [comment]
let's get this in ?

**Pull request merged**